frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureStrategyMenuCard/FeatureStrategyMenuCard.tsx

```diff
@@ -86,6 +88,9 @@ export const FeatureStrategyMenuCard = ({
                 buttonTitle: strategy.displayName || strategyName,
             },
         });
+        if (onClose) {
+            onClose();
+        }
```

Comment:
Not important, but we can simplify this to:
```suggestion
        onClose?.()
```